src/L2_projection.jl

```diff
@@ -3,24 +3,48 @@ abstract type AbstractProjector end
 
 struct L2Projector <: AbstractProjector
     fe_values::CellValues
-    M_cholesky # ::SuiteSparse.CHOLMOD.Factor{Float64}
-    dh::DofHandler
+    M_cholesky #::SuiteSparse.CHOLMOD.Factor{Float64}
+    dh::MixedDofHandler
```

Comment:
Does it still work with the regular DofHandler?

Reply:
The MixedDofHandler is constructed internally, so I think there is no need for it to offer an alternative with the regular DofHandler. The old tests still work with the new version (you can do all computations with DofHandler and then still use this for interpolating your results in the end). Do you see any other reason to have another version with the regular DofHandler?

Reply:
:+1: